shop-be/product-service/src/libs/lambda.ts

```diff
@@ -1,6 +1,13 @@
 import middy from "@middy/core"
 import middyJsonBodyParser from "@middy/http-json-body-parser"
+import validator from '@middy/validator'
 
-export const middyfy = (handler) => {
-  return middy(handler).use(middyJsonBodyParser())
+// const logger = {
```

Comment:
Why do you comment logger?

Reply:
used it for debugging, I don't think it can be useful in my implementation))

Reply:
ok